include/aspect/initial_temperature/function.h

```diff
@@ -78,6 +80,13 @@ namespace aspect
          * A function object representing the temperature.
          */
         Functions::ParsedFunction<dim> function;
+
+        /**
+         * The coordinate representation to evaluate the function. Possible
+         * choices are depth, cartesian and spherical.
+         */
+        ::aspect::Utilities::Coordinates::CoordinateSystem coordinate_system;
```

Comment:
I tried, and it didn't work. It didn't give a compilation error, it lost functionality. Specifically it didn't recognize set Coordinate system.